ITEST/itest.smoke/test-resources/simple_mul.json.gzip.post.req

```diff
@@ -0,0 +1,9 @@
+POST /REST/simple/mul HTTP/1.1
+Content-Type: application/json
+Accept: application/json;q=1.0, */*;q=0.8
+Accept-Encoding: gzip
+
+{
+  "x": 7,
+  "y": 3
+}
```

Comment:
end of line?

Reply:
Fixed, because of this test data is tolerant to whitespaces